README.md

```diff
@@ -1,31 +1,30 @@
-# ![nf-core/rnafusion](https://raw.githubusercontent.com/nf-core/rnafusion/master/docs/images/rnafusion_logo.png)
 
-[![Build Status](https://travis-ci.org/nf-core/rnafusion.svg?branch=master)](https://travis-ci.org/nf-core/rnafusion)
-[![Nextflow](https://img.shields.io/badge/nextflow-%E2%89%A50.32.0-brightgreen.svg)](https://www.nextflow.io/)
+# ![nf-core/rnafusion](docs/images/nf-core-rnafusion_logo.png)
+
+**Nextflow rnafusion analysis pipeline, part of the nf-core community.**.
```

Comment:
I'm not sure that `part of the nf-core community` is what best describe the pipeline

Reply:
What about this?
RNA sequencing analysis pipeline with curated list of tools for detecting and visualizing fusion genes

Reply:
Wonderful, I love it

Reply:
Done

Reply:
```suggestion
**RNA sequencing analysis pipeline with curated list of tools for detecting and visualizing fusion genes**.
```